### DISCUSSION TITLE
Paginate Project listing page

### DISCUSSION THREAD

**Discussion by godwinnwalozie** [jitinnair1/gradfolio#19] [Q&A] [answered]
Hello , I am using Jekyll for the first time and I love it.
Please I would love to
1. paginate  the project page, just like the blog post (6 projects or 9 projects on a page).  How can i acheive this .
2. Add a search box on the project page.
Thank you for your wonderful theme

**Comment by jitinnair1** [comment, marked as answer]
Hi @godwinnwalozie,

Implementing both of these features is a bit involved. However, If you would like to implement these you can search for Jekyll plugins that can add this functionality (Check out [this page](https://blog.webjeda.com/instant-jekyll-search/) for information on search and [this one](https://gist.github.com/Phlow/5613fb3f18946f577f071e2a258749a3) on information on pagination)

On the pagination bit, you will have to combine the method used for blog posts in the theme (please check `blog/index.html`)  with the approach used in the link above to get pagination on the Projects page.

Thank you for initiating this discussion. I will consider adding such functionality in later releases.
